content/en/apps/examples/anc.md

```diff
@@ -1,16 +1,55 @@
 ---
-title: "Antenatal Care Reference App"
-linkTitle: "ANC Reference App"
+title: "Antenatal Care"
+linkTitle: "Antenatal Care"
```

Comment:
Given that the config includes PNC, I think we should actually be referring to this as a _Maternal and Newborn Health_ app instead. What do you think?

Reply:
sounds good, updated throughout the text